app/components/menu-sidebar-item.js

```diff
@@ -6,8 +6,8 @@ const MenuSidebarItemComponent = Component.extend({
   minor: false,
   actions: {
     closeAfterNavigation() {
-      if (this.minor) {
-        this.layoutManager.closeProfileMenu();
+      if (this.minor && this.layoutManager.profileMenuOpen) {
```

Comment:
I know I suggested this myself, but maybe it is better to add the `closeProfileMenu` method again, since now we have to check the status of the profileMenu here before toggling, otherwise we do not have to care about the state of the service and just say close.

Reply:
Okay, I'll add it back 👍🏼